### PR TITLE
Structuring CSS on the File System

### PR DESCRIPTION
**Pull request by mguterl**
@mitchlloyd @rockwood @joelturnbull any feedback you have would be appreciated.

### CODE REVIEW

File: posts/2014-03-27-structuring-css-on-the-filesystem.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+```sass
+.text-left
+  text-align: left !important
+  
+.flush
+  margin: 0 !important
+```
+
+[https://github.com/csswizardry/inuit.css/blob/master/generic/_helper.scss]()
```
Comment (joelturnbull) [inline posts/2014-03-27-structuring-css-on-the-filesystem.md]: did you mean [https://github.com/csswizardry/inuit.css/blob/master/generic/_helper.scss](https://github.com/csswizardry/inuit.css/blob/master/generic/_helper.scss)

**Comment by joelturnbull** [comment]
Looks good. I don't have a dog in the fight, it makes sense and I'd be willing to get on board with it. I too am tired of resorting to text search all the time.

**Comment by mguterl** [comment]
@joelturnbull - aside from the additional feedback I solicited from @mitchlloyd about nesting naked elements, I believe I'm ready to call this post "done". I'm happy to keep iterating on it if there are places we want to improve, but I wanted to let you know where it stands.

**Comment by mguterl** [comment]
@joelturnbull - this is ready unless you have other edits to suggest.

**Comment by joelturnbull** [comment]
@mguterl done thanks for getting this submitted early

**Comment by rockwood** [comment]
This looks really great!

Just a couple quick points: 
- The folder structure should probably use `.sass` extensions to match the rest of the examples.
- I would recommend the helper `text--left` instead of `text-left` to match the BEM modifier syntax.

**Comment by mguterl** [comment]
Thanks @rockwood. I just incorporated your comments.

**Comment by cdmwebs** [comment]
The only thing I'd offer here is that I like to use the underscore prefix on anything meant to be included. It's just another signal to someone else that this file is not meant to be included in a layout, but mixed in to a manifest.

**Comment by mguterl** [comment]
@cdmwebs - are you suggesting that we should name files that contain mixins with an underscore too?

**Comment by cdmwebs** [comment]
Anything that isn't a manifest. Sass won't compile partials, so it helps to explain that this file is meant to be mixed in.

**Comment by rockwood** [comment]
Yeah, I'm gonna go ahead and have to disagree with you there. Having to
prefix nearly every sass file with an _ seems lame, and it's inconsistent
with the way we name JS files. Rails will only compile whats required in
application.sass so it's not that it's adding additional compile time.

-Kevin

**Pull request merged**